build/ci/set-up-ops-manager.sh

```diff
@@ -48,21 +48,33 @@ echo "generate password for owner"
 password=$(date +%s | sha256sum | base64 | head -c 8)0_
 
 echo "generate email for owner"
-MCLI_PUBLIC_API_KEY=$(date +%s | sha256sum | base64 | head -c 8)@ops-manager-team
-export MCLI_PUBLIC_API_KEY
+email=$(date +%s | sha256sum | base64 | head -c 8)@ops-manager-team
+
 
 echo "create first user"
-MCLI_PRIVATE_API_KEY=$(./bin/mongocli om owner create --firstName evergreen --lastName evergreen --email "${MCLI_PUBLIC_API_KEY}" --password "${password}" --accessListIp "127.0.0.1/1" -o="go-template={{.APIKey}}")
-export MCLI_PRIVATE_API_KEY
+cat <<EOF > credentials.tmpl
+#!/bin/bash
+
+set -euo pipefail
+
+export MCLI_PUBLIC_API_KEY={{.ProgrammaticAPIKey.PublicKey}}
+export MCLI_PRIVATE_API_KEY={{.ProgrammaticAPIKey.PrivateKey}}
```

Comment:
[q] I don't see where `.ProgrammaticAPIKey.PublicKey` is coming from

Reply:
this is a template the command will output as (see line 64 `-o=go-template-file=`, the value will come from the api response https://docs.opsmanager.mongodb.com/current/reference/api/user-create-first/

Reply:
Thank you very much! I have missed how we were using this file. Great job on this!! 💯 🥇